.eslintrc

```diff
@@ -1,8 +1,4 @@
 {
-  "extends": [
-    "eslint-config-unjs"
-  ],
-  "rules": {
-    "no-redeclare": "off"
-  }
```

Comment:
ESLint passes without this rule defined, so I deleted it.
Please let me know if you need for some reasons.

Reply:
Updated: I confirmed `"no-redeclare": "off"` is included in `eslint-config-unjs`, by running `pnpm eslint --print-config ./src/index.ts > test.json`

```txt
...
"no-redeclare": [
  "off",
  {
    "builtinGlobals": false
  }
],
...
```